### PR TITLE
Created a scroll to top Button

### PR DESCRIPTION
**Pull request by anujji1999**
Fixed issue: #80 
Created a scroll to top button by making changes in the index.html file

After changes:-
after

### CODE REVIEW

**Comment by jywarren** [comment]
This looks good, but do you think you could name it more descriptively, so it's easier to see what the PR is about? This is very helpful to our reviewing team. Thanks!!!

**Comment by Rishabh570** [comment]
@anujji1999 Yes, please try to be descriptive in your commits/PR title...also it would be great if you share a working GIF of your changes :)
Thanks for your contribution :+1:

**Comment by anujji1999** [comment]
Thanks @jywarren @Rishabh570 for your comments.
I am trying to make it more descriptive.

**Comment by anujji1999** [comment]
@jywarren @Rishabh570 @DakshGondaliya @ajzliu @pabisovsky LGTM.
Please review.

**Comment by Rishabh570** [comment]
@anujji1999 Great!!! I think this is okay for now, but it would be great if the transition is smooth and not jerky (just some thoughts)...
Overall, looks good :tada:

**Comment by anujji1999** [comment]
@Rishabh570 Sorry, i didn't get it.

**Comment by anujji1999** [comment]
@Rishabh570 please merge my pr.
or mention others to merge it.

**Comment by anujji1999** [comment]
@jywarren @SidharthBansal @DakshGondaliya I have already created this pull request.
Please see this.

**Comment by nonas-hunter** [comment]
@anujji1999 if you are interested in making it scroll more smoothly, an easy solution would to replace the current onclick function you have in the html file with the following:

`onclick="window.scroll({ top: 100, left: 100, behavior: 'smooth' });"`

**Comment by anujji1999** [comment]
Done
@Rishabh570 @jywarren @SidharthBansal @DakshGondaliya 
Please reveiw it.
LGTM.

**Comment by Rishabh570** [comment]
@anujji1999 Two things,
- We have `demo.css` for styles and `community-toolbox.js` for scripts (it is the main one). It would be great if you put your styles and js in those files instead of putting them in `index.html`.
- Please provide a working GIF of your work to let us know the outcome of your changes, it requires extra effort to fix things later.

Thanks for your contribution :+1:

**Comment by anujji1999** [comment]
Done
@Rishabh570 @jywarren @SidharthBansal @DakshGondaliya 
Please reveiw it.
LGTM.

**Comment by Rishabh570** [comment]
@anujji1999 Changes look good, can you share a GIF of your recent changes...?

**Comment by anujji1999** [comment]
Hello @Rishabh570 
Actually, I am still not able to link the external JS into the index.html file.
Can you please help in personal?

**Comment by Rishabh570** [comment]
@anujji1999 Here is what you need to do in `index.html` and `community-toolbox.js`,

**index.html**

screenshot from 2018-11-15 09-37-55

You were calling the function wrongly...do this in `index.html` in the shown place only...

**community-toolbox.js**

screenshot from 2018-11-15 09-38-19

(You need to export the `scrollFunction` function also)

Let me know if you don't understand anything here...

**Comment by anujji1999** [comment]
@Rishabh570 it is still not working.

**Comment by anujji1999** [comment]
Done @Rishabh570 
Please check it.

File: index.html [hunk 1]
```diff
@@ -9,17 +9,24 @@
...
-
+	 
+    <link rel='stylesheet' href='https://use.fontawesome.com/releases/v5.5.0/css/all.css' integrity='sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU' crossorigin='anonymous'>
+   
     <script src="node_modules/jquery/dist/jquery.min.js"></script>
     <script src="node_modules/bootstrap/dist/js/bootstrap.min.js"></script>
     <script src="node_modules/chart.js/dist/Chart.js"></script>
     <script src="node_modules/urlhash/urlHash.js"></script>
     <script src="dist/community-toolbox.js" charset="utf-8"></script>
+    <script type="text/javascript" src="src/community-toolbox.js"></script>
```
Comment (Rishabh570) [inline index.html]: Please remove this line, please...that is imported just above it from `dist`...

**Comment by Rishabh570** [comment]
Hi @anujji1999, are you still stuck on it?

**Comment by anujji1999** [comment]
Due to exams, I was not getting time to work on it.I will definitely work on it tommorow.

**Comment by anujji1999** [comment]
Hi, @Rishabh570 . I am not able to run grunt build. Can you please help me?

**Comment by Rishabh570** [comment]
@anujji1999 Yeah sure, first make sure you've done `npm install` because grunt is installed by it as a dev-dependency then run `grunt build` after which your changes (css or js changes) will take effect...

BTW what error is showing when you run grunt build?
Thanks!

**Comment by anujji1999** [comment]
@Rishabh570 How to do npm install?

**Comment by anujji1999** [comment]
Actually i am new to open source. So, i don't know much about it.

**Comment by Rishabh570** [comment]
Go to the project folder and run `npm install` in the terminal, like this,

screenshot from 2018-12-22 22-06-52

**Comment by anujji1999** [comment]
@Rishabh570 I am a windows user.
So, i have to do this in cmd or git bash?

**Comment by anujji1999** [comment]
@Rishabh570 Please check now.

**Comment by anujji1999** [comment]
@Rishabh570 LGTM

**Comment by anujji1999** [comment]
@jywarren Please check this and merge my PR.

**Comment by subhahu123** [comment]
@anujji1999  Resolve your conflicts  and 
Also , squash your commits

**Comment by namangupta01** [comment]
screen shot 2019-01-03 at 12 48 47 am
Why it is showing so much of line deleted and added?

**Comment by Rishabh570** [comment]
Hi @namangupta01, I guess as this PR includes many commits (redundant somehow) that might be the reason...@anujji1999 might tell?

**Comment by gauravano** [comment]
Hi, the scroll to top is implemented in #140 so closing this. Thanks for your help and we really appreciate your efforts. We would love to have your help with other issues. Thanks!

**Pull request closed without merging**